peerdid/core/utils.py

```diff
@@ -0,0 +1,33 @@
+import base64
+import json
+
+
+def _is_json(str_to_check: str) -> bool:
+    """
+    Checks if str is JSON
+    :param str_to_check: sting to check
+    :return: true if str is JSON, otherwise raises ValueError or TypeError
+    :raises TypeError: if str_to_check is not str type
+    :raises ValueError: if str_to_check is not valid JSON
+    """
+    json.loads(str_to_check)
```

Comment:
I think makes sense to either catch exceptions here and return False or rename to `check` (or `validate`) since `is` implies some bool, here we return or fail